Feature: Timesheet tab

cristian-bts opened the pull request:
![New Mockup 2](https://user-images.githubusercontent.com/42077660/122232890-375ee100-ce81-11eb-9b86-1d6b41ef7dbe.png)

cristian-bts committed:
The pod of calendar was added

cristian-bts committed:
calendar week view was added

manuelsalinas-legion commented:
@cristian-bts 
<img width="667" alt="iPhone_12_and_TeamViewController_swift" src="https://user-images.githubusercontent.com/42077660/122788417-ee8d9a80-d27b-11eb-914f-4ba15fe0f20b.png">

cristian-bts committed:
Uploading last changes, calendar does not work

manusalinas-bts committed:
Localization files, renaming fixes, ui fixes and added scripts for localization

cristian-bts committed:
saving calendar chagnes to pull new changes

cristian-bts committed:
Solvi g conflicts

cristian-bts committed:
Adding relations between storyboard and timesheet view controller

manusalinas-bts committed:
Nice week view

cristian-bts committed:
Merge branch 'features/TimesheetTab/master' of github.com:manuelsalinas-legion/BTSPocket into features/TimesheetTab/master

manusalinas-bts committed:
BTSPocket: Base Controller for new  TS

manusalinas-bts committed:
Missing icons

cristian-bts committed:
The timesheet table view was added, saving to pull changes

cristian-bts committed:
Merge changes

cristian-bts committed:
Timesheet view cell was added

manusalinas-bts committed:
fixes on timesheet cell

cristian-bts committed:
Save changes to pull changes, the alert controller was added but is not functional

cristian-bts committed:
Merge branch 'features/TimesheetTab/master' of github.com:manuelsalinas-legion/BTSPocket into features/TimesheetTab/master

cristian-bts committed:
Upload last changes

manusalinas-bts committed:
textview cell, pod to avoid keyboard (https://github.com/hackiftekhar/IQKeyboardManager) and fixes on UI

manusalinas-bts committed:
Notes Controller

manusalinas-bts committed:
Fixing loading on Login

cristian-bts committed:
Saving setting in the detail timesheet controller to pull changes

cristian-bts committed:
Merge changes

cristian-bts committed:
Obtain projects since profile

cristian-bts committed:
add changes in get projects function

manusalinas-bts committed:
- Project section added
- Table detail project now is bidimensional by section and row
- UI fixed on cells
- Selected color
- Separators fixed
- Picker cell
- Collapsable behavior

manusalinas-bts committed:
Login background

cristian-bts committed:
create and update view was done without initegration with backend

cristian-bts committed:
Merge changes

cristian-bts committed:
pass project name to update view

manusalinas-bts committed:
App icons (incomplete)

manusalinas-bts committed:
missing icons added

cristian-bts committed:
the localized were added in the strings

cristian-bts committed:
Merge branch 'features/TimesheetTab/master' of github.com:manuelsalinas-legion/BTSPocket into features/TimesheetTab/master

manusalinas-bts committed:
Biometrics icons

cristian-bts committed:
Save changes to pull branch

cristian-bts committed:
Merge branch 'features/TimesheetTab/master' of github.com:manuelsalinas-legion/BTSPocket into features/TimesheetTab/master

cristian-bts committed:
icons was added in touch and face id, parameter boolean was added in logout function
